Dockerfile

```diff
@@ -53,9 +53,6 @@ COPY --from=build --chown=appuser:appgroup \
 COPY --from=build --chown=appuser:appgroup \
         /app/node_modules ./node_modules
 
-COPY --from=build --chown=appuser:appgroup \
```

Comment:
this leaked in from elsewhere. but it might as well get merged. this COPY is superfluous.